azure-mgmt-graph-rbac/src/main/java/com/azure/management/graphrbac/implementation/ActiveDirectoryUsersImpl.java

```diff
@@ -69,22 +70,18 @@ public ActiveDirectoryUserImpl getByName(String upn) {
     @Override
     public Mono<ActiveDirectoryUser> getByNameAsync(final String name) {
         return getManager().getInner().users().getAsync(name)
```

Comment:
I think this line will have Error when not found. So it need to be concated with `onErrorResume`?